src/components/CollectionsPage/CollectionsPage.tsx

```diff
@@ -29,7 +29,7 @@ import CollectionRow from './CollectionRow'
 import { Props, TABS } from './CollectionsPage.types'
 import './CollectionsPage.css'
 
-const PAGE_SIZE = 20
+const PAGE_SIZE = 6
```

Comment:
doesn't this make the second row cut in half? 
<img width="1129" alt="Screen Shot 2022-05-16 at 18 54 42" src="https://user-images.githubusercontent.com/2781777/168688796-1e040ed3-1917-43f8-9318-94348eeacbad.png">

(disregard the broken images, that's only on my local env)

Reply:
Oops, I forgot to get that back to `20`, I was testing locally with a smaller page size. I put it back to 20!